app/javascript/components/pages/UserProfile.js

```diff
@@ -1,24 +1,88 @@
 import React, { Component } from "react";
-import { fetchProfile } from "../api";
+import { thead, tr, th, tbody, Table } from "reactstrap";
```

Comment:
I can see you getting Table from reactstrap, but thead, tr, th, tbody are all regular HTML tags. Do they need to be imported?

Reply:
Good catch. Will be removed before next update